orm/xorm/xorm_parallel.go

```diff
@@ -76,3 +83,42 @@ func (xpr *XormParallel) FindParallelByCondition(listPtr interface{}, cond FindC
 
 	return nil
 }
+
+// CountParallelByCondition executes SELECT COUNT(*) query to all of the shards with conditions
+func (xpr *XormParallel) CountParallelByCondition(objPtr interface{}, cond FindCondition) ([]int64, error) {
+	vt := reflect.TypeOf(objPtr)
+	if vt.Kind() != reflect.Ptr {
+		return nil, errors.NewErrArgType("objPtr must be a pointer")
+	}
+
+	// create session with the condition
+	slaves := xpr.orm.Slaves(cond.Table)
+	sessions := xpr.CreateSessionsWithCondition(cond)
+
+	// execute query
+	var errList []error
+	results := make(chan int64, len(slaves))
```

Comment:
@yyoshiki41  `len(slaves)` correspond to `len(sessions)`, so don't need to use `slaves` in this method.